mapping-for-navigation/_posts/0000-01-09-modeling-interesections-for-map-navigation.md

```diff
@@ -0,0 +1,191 @@
+  [fd18165a]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_error.png "Incorrect drawing of the three way T-shape dual carriageway and bi-directional roads intersection (fullsize view)"
+  [1f05b9a7]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_wireframe.png "Three way T-shape dual carriageway and bi-directional roads intersection (wireframe view)"
+  [595e07f7]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_wireframe_error.png "Incorrect drawing of the three way T-shape dual carriageway and bi-directional roads intersection (wireframe view)"
+
+-   This place ☝️ [on the map](https://www.openstreetmap.org/edit#map=19/33.95899/-118.18513) 🗺
+
+The same as for intersection of a dual carriageway and bi-directional roads, no need to squeeze everything into one node bow-tie intersection to have sausage roads. Keep ways that represnt dual carriageway road straight. Draw internal edge of the bi-directional road between ways of dual carriageway lines on the intersection area parallel to one of the pedestrian crossing paths to form right intersection geometry.
+
+| ✅ **Recommended**  | ❌ **Not recommended** |
+|---|---|
+| [![][fae561f7]][fae561f7] | [![][902f4aee]][902f4aee] |
+| (_[fullsize][fae561f7]_ \| _[wireframe][641c8040] view_) | (_[fullsize][902f4aee]_ \| _[wireframe][e606f9b6] view_) |
+
+  [fae561f7]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_double_point_1.png "Three way T-shape dual carriageway and bi-directional roads intersection (fullsize view)"
+  [902f4aee]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_double_point_err_1.png "Incorrect drawing of the three way T-shape dual carriageway and bi-directional roads intersection (fullsize view)"
+  [641c8040]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_double_point_wireframe_1.png "Three way T-shape dual carriageway and bi-directional roads intersection (wireframe view)"
+  [e606f9b6]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_double_point_wireframe_err_1.png "Incorrect drawing of the three way T-shape dual carriageway and bi-directional roads intersection (wireframe view)"
+
+-   This place ☝️ [on the map](https://www.openstreetmap.org/edit#map=19/33.70194/-117.86151) 🗺
+
+### 6. 3-way T-shape intersection (dual carriageway and bi-directional roads single point ⤫)
+
+| ✅ **Recommended**  | ❌ **Not recommended** |
+|---|---|
+| [![][3178c41e]][3178c41e] | [![][e20d4278]][e20d4278] |
+| (_[fullsize][3178c41e]_ \| _[wireframe][b37286cb] view_) | (_[fullsize][e20d4278]_ \| _[wireframe][050b1dfe] view_)  |
+
+  [3178c41e]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_single_point.png "Three way T-shape dual carriageway and bi-directional roads single point intersection (fullsize view)"
```

Comment:
This example happens to show a median barrier, which reminds me: all the dual carriageway examples in this guide show a physical barrier separating the turn lane from opposing traffic. However, in some areas, it’s common for there to be just a painted double yellow line, such that every turn lane represents a brief break in the median:

[<img width="838" alt="tylersville" src="https://user-images.githubusercontent.com/1231218/35139749-5964fd64-fca9-11e7-8b55-eb7f75be27dc.png">](http://www.openstreetmap.org/#map=19/39.34418/-84.29940)

In this case, I’ve chosen to keep things simple and continue the dual carriageway as though the median continues right up to the intersection. Is this preferable, or should a mapper begin a new dual carriageway after every intersection?